feathr_project/feathr/definition/anchor.py

```diff
@@ -37,7 +37,7 @@ def __init__(self,
 
     def validate_features(self):
         """Validate that anchor is non-empty and all its features share the same key"""
-        assert len(self.features) > 0
+        # assert len(self.features) > 0
```

Comment:
Why does this line comment out?

Reply:
> Why does this line comment out?

If the anchor is fetched directly from Purview, it will only be a node, we need to fetch its connected edges to get self.features. 
If the anchor is generated locally, (I mean passing all params in its constructur). it will have .features. 
Comment this out to make it work for both scenarios.